File: Helló, Liskov!/Szülő-gyerek/szulogyerek.java
class szulo
 {
   protected int kor;
   protected String nev;
	
   public void setNev(String neve)
   {
	nev = neve;
   }

}

class gyerek extends szulo
 {
	public String getNev() 
	{
		return nev;
    }
 }   

class szuloGyerek
{
 public static void main (String args[])
{
	szulo l = new gyerek();
	l.setNev("szülő");
	gyerek k = new gyerek();
	k.setNev("gyerek.");
	System.out.println(k.getNev() + " " + l.getNev());
 }
}

File: Helló, Liskov!/Szülő-gyerek/szulogyerek.cpp
#include <iostream>

using namespace std;

class Szulo
{

};
    
class Gyerek: public Szulo
{
    void viselkedik()
    {
	   cout<<"Viselkedek!";	
	}
};
    
int main ()
{
    Szulo* sz= new Gyerek;
   
    cout<<sz->viselkedik();      
}

